in_toto/in_toto_mock.py

```diff
@@ -91,8 +96,8 @@ def main():
 
   args = parser.parse_args()
 
-  # Turn on all the `log.info()` in the library
-  log.logging.getLogger().setLevel(log.logging.INFO)
+  # Default to verbose
```

Comment:
I don't know how descriptive this comment is. I personally find it a little confusing.

Reply:
I can rephrase, e.g.:
*in-toto-mock should not be used to secure the supply chain but only to try out in-toto-run, hence it gives verbose user feedback*